src/main/java/com/mongodb/hibernate/internal/type/ValueConversions.java

```diff
@@ -72,6 +93,18 @@ public static BsonBoolean toBsonValue(boolean value) {
         return BsonBoolean.valueOf(value);
     }
 
+    /**
+     * <a
+     * href="https://docs.jboss.org/hibernate/orm/6.6/userguide/html_single/Hibernate_User_Guide.html#basic-character">
```

Comment:
It might be a little bit better to avoid Hibernate ORM version in the doc for it is a moving target. Using `current` in lieu of version usually serve the same purpose.
```
https://docs.jboss.org/hibernate/orm/current/userguide/html_single/Hibernate_User_Guide.html#basic-character
```